en/docs/get-started/quick-start-guide/integration-qsg.md

```diff
@@ -224,4 +224,4 @@ Here you will not see an onboarding page but a listing of the deployed services
 ## What's next?
 
 - [Develop your first integration solution]({{base_path}}/integrate/develop/integration-development-kickstart).
-- Try out the **tutorials** and **examples** available in the [Learn section of our documentation]({{base_path}}/integrate/learn-overview/#integration-use-cases).
+- Try out the **tutorials** and **examples** available in the [Integrate section of our documentation]({{base_path}}/integrate/learn-overview/#integration-use-cases).
```

Comment:
Tutorials are not in the 'Integrate' section

Reply:
The tutorials list is available here right? We don't have a list of them anywhere else.

Reply:
For now I will remove mention of tutorials until the scenario tutorials come in.